js/main.js

```diff
@@ -208,64 +208,46 @@ function StravaEnhancementSuite($, options) {
 
   // Methods //////////////////////////////////////////////////////////////////
 
-  // Own activity pages
+  // Own activity page
   $.always(function() {
     // Detect whether we are viewing our own activity and to activate the dialog
-    var edit_activity = $('.sidenav .edit-activity');
-
-    // Seems to be truncated in size
-    $('#heading span.title')
-      .css('display', 'inline')
-      ;
+    var edit_activity = $('a[title="Edit this activity"]');
 
     if (edit_activity.length === 0) {
       return;
     }
 
-    // Clicking the activity name launches the dialog
-    $('#heading h2.activity-name')
+    // Edit activity by clicking on it's name
+    $('.activity-name')
       .css('cursor', 'pointer')
       .on('click', function() {
-        edit_activity.click();
+        location.pathname = edit_activity.attr('href')
       })
-      ;
 
-    // Enter whilst editing the name saves the dialog
-    $('body').on('keydown', '.lightbox.edit_activity input[type=text]', function (e) {
-      if (e.keyCode === 13) {
-        $(this)
-          .parents('.lightbox')
-          .find('input[type=submit]')
-          .click()
-          ;
-      }
-    });
+    // Edit activity by pressing "e"
+    $('body').on('keydown', function () {
```

Comment:
First feature, not just a bugfix :D 🚀 

Edit activity by pressing "e", how it's done on GitHub, GitLab, Jira, ...